persistence/src/main/java/org/cbioportal/genome_nexus/persistence/internal/EnsemblRepositoryCustom.java

```diff
@@ -39,4 +41,5 @@
     EnsemblTranscript findOneByHugoSymbolIgnoreCase(String hugoSymbol, String isoformOverrideSource);
     EnsemblGene getCanonicalEnsemblGeneIdByHugoSymbol(String hugoSymbol);
     EnsemblGene getCanonicalEnsemblGeneIdByEntrezGeneId(String ensemblGeneId);
+    HashMap<String, String> getHugoSymbolToEntrezGeneIdMap();
```

Comment:
It might make sense to map to the whole `EnsemblCanonical` transcript instead of only `EntrezGeneId`. That way you can get the other info about the canonical transcript in a quick way as well if you would want to. Anyway I know this PR is a quick fix, so prolly better to leave it as is for now